### PR TITLE
(Bug) TW 2.0 doesn't work with Ganache-cli 

### LINKED ISSUE #1162: (Bug) TW 2.0 doesn't work with Ganache-cli
If you are reporting a problem with Token Wizard, please include the following information:

### Which network did you use? (Mainnet, Kovan, Rinkeby, etc.)

Localhost:8545

### If you were able to create it, what is the URL of your crowdsale?

No, I was not able to deploy crowdsale. I was trying to create Minted capped crowdsale with Ganache without using any wallet. There are two problems here:
1. I see warning messages, that wallet is not installed.
2. I was not able to create crowdsale. 1st transaction to deploy Proxy contract failed. Log is attached.

Additional details:
- I've started TW2.0 with `npm run dev:minted`
- `./public/contracts` folder comprises of all needed assets: binary code, ABI and source code for Proxy.

### Do you see errors in the dev console? If yes, please include a screenshot

To open the dev console in Google Chrome, press F12, or go to `View -> Developer -> Developer Tools`, and open the Console tab

If you see errors, please right click on them and "Save as..". Zip saved file and attach it to the Issue.

[localhost-1538136803528.zip](https://github.com/poanetwork/token-wizard/files/2428102/localhost-1538136803528.zip)

---

### PR DESCRIPTION
**Pull request by mariano-aguero**
Closes #1162

How to test it:
Try to reproduce the problem described in the related issue

### CODE REVIEW

File: package-scripts.js [hunk 1]
```diff
@@ -24,13 +24,7 @@ module.exports = {
...
-        'git submodule update --init --recursive --remote',
-        'npm run installWeb3',
-        'bash ./scripts/generateProxyContracts.sh',
-        'npm run moveTruffleConfigPublic',
-        'npm run moveSolcVersionOutputPublic'
-      ),
+      default: series('git submodule update --init --recursive --remote', 'npm run installWeb3'),
       Minted: {
         default: series(
           'nps test.e2e.stop',
```
Comment (fernandomg) [inline package-scripts.js]: @mariano-aguero is it possible that in a clean installation, web3 is missing?

I added the line npm run installWeb3 and everything worked fine.

(same with `Dutch`)

**Comment by fernandomg** [comment]
@mariano-aguero
I'm having the 'Warning' modal appearing on every page transition (when step1, step2, etc loads, the modal appears), is it expected?

File: package-scripts.js (hunk 1 above)
Comment (mariano-aguero) [reply package-scripts.js]: @fernandomg I think that the default script that includes the installation of web3 is executed first and then the dev:minted or dev:dutch script.
Anyway, I move the script to make it functional

**Comment by mariano-aguero** [comment]
@fernandomg 

> I'm getting the 'Warning' modal for the no-wallet available appearing on every page transition (when step1, step2, etc loads, the modal appears), is it expected?

Yes, because there is no wallet and it is the previous behavior

Regards

**Comment by mariano-aguero** [comment]
@vbaranov @dennis00010011b @fernandomg 

Removed warnings on development enviroment, and in the contribution page

Regards

**Comment by dennis00010011b** [comment]
Tested it. No issues were found

**Pull request merged**